packages/network/src/dev/emitter.ts

```diff
@@ -0,0 +1,27 @@
+import Emittery from "emittery";
+import { TableId } from "@latticexyz/utils";
+
+// Expose a global emitter to ease in getting data from our internals to the dev UI
+
+export type EmitterEvents = {
+  storeEvent: {
+    chainId: number;
+    worldAddress: string;
+    transactionHash: string;
+    blockNumber: number;
+    logIndex: number;
+    event: "StoreSetRecord" | "StoreSetField" | "StoreDeleteRecord";
```

Comment:
should we add `StoreEphemeralRecord` here too? (Should we import this from `common/storeEvents` to not have to maintain it separately?)

Reply:
yes and yes, this PR started before that feature was finalized/merged :)